packages/plugin-did-you-mean/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# @shopify/plugin-did-you-mean
+
+## 3.43.0
```

Comment:
Not sure how versioning is done in CLI. Is this the version we want? Was added manually 😄

Reply:
The changelog is generated automatically:
When you do `pnpm changeset add`, a changeset file will be created, when merged, `changeset` will automatically update the changelog.